### DISCUSSION TITLE
donate button?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [fluxcd/flux2#1502] [Q&A] [answered]
how i can thanks with some USD for the stuff?

**Comment by hiddeco** [comment, marked as answer]
Hi @dzmitry-lahoda,

Thank you for your interest in donating! We do at this time not accept direct donations. Most of the work done on Flux is funded by companies donating people to work on the project (like myself), or people making contributions in their spare time (for which we are very thankful!).

As far as I am aware, the CNCF (which funds some of the services we make use of) is partly funded through the Linux Foundation, so [donating to them](https://www.linuxfoundation.org/en/about/donate/) would always be an option.

However, to directly support our work and help us, you have a couple of options:
1. You could add your organization to [the adopters list](https://fluxcd.io/adopters)
2. Help us out with testing, documentation and/or code contributions
3. Provide support to other users in our [discussions](https://github.com/fluxcd/flux2/discussions) or [on Slack](https://cloud-native.slack.com/archives/CLAJ40HV3)

Thank you! :sunflower:

**Reply by kingdonb** [reply]
Another way that you can support Flux, though indirectly (as I'm fairly certain we don't receive financial benefit from sales, and I'm equally skeptical the CNCF makes any money on the store) is by purchasing Flux merch from the CNCF store, and wearing it while representing the Flux community positively in any public setting:

https://store.cncf.io/collections/flux

Thanks for your enthusiasm @dzmitry-lahoda !

**Reply by dzmitry-lahoda** [reply]
donating to cncf is good, add our org as adopter later hope our org will grow big)) and we are doing  agones multicluster flux.
